Makefile

```diff
@@ -87,25 +87,6 @@ train_sector_classifier:
 train_instruments_classifier:
 	poetry run python classifiers/trainer.py --argilla-dataset-name policy-instrument-text-classifier
 
-merge_metadata:
```

Comment:
Out of interest does this mean we aren't merging metadata anymore?

Reply:
explorer now does it by providing the metadata CSV as input: https://github.com/climatepolicyradar/explorer/pull/45/files